### PR TITLE
[Arista NG Firewall] Bug fixes for initial release

### LINKED ISSUE #6853: [Arista NG Firewall] Couple bugs found in initial release
This initial release had a few issues:

1. Manifest had errors in the variables for the interface names and aliases under the TCP input. 
  a. Interface alias 0 should have been 1
  b. Interface alias 1 should have been 2
2. Agent HBS configurations have duplicated `_conf` fields
  a. Causing `duplicated mapping key at line 11, column -38: _conf: ^` error when attempting to apply configurations

I will be creating a PR shortly to correct these.

### PR DESCRIPTION
**Pull request by MakoWish**
## Type of Change
- Bug

## What does this PR do?

* Corrects variables in manifest
* Fixes duplicate assignment of `_conf` field

## Checklist

- [X] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [X] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [X] Correct variables in TCP input section of manifest
- [X] Fix duplicate assignment of `_conf` field in both `tcp.yml.hbs` and `udp.yml.hbs` agent configurations. 

## Related issues

- Closes #6853

### CODE REVIEW

**Comment by MakoWish** [comment]
Pinging @efd6 .

**Comment by efd6** [comment]
```diff
diff --git a/packages/arista_ngfw/data_stream/log/fields/ecs.yml b/packages/arista_ngfw/data_stream/log/fields/ecs.yml
index 63ac7c32b..26c4196ed 100755
--- a/packages/arista_ngfw/data_stream/log/fields/ecs.yml
+++ b/packages/arista_ngfw/data_stream/log/fields/ecs.yml
@@ -240,8 +240,6 @@
   name: source.geo.country_iso_code
 - external: ecs
   name: source.geo.country_name
-- external: ecs
-  name: source.geo.dma_code
 - external: ecs
   name: source.geo.location
 - external: ecs
diff --git a/packages/arista_ngfw/docs/README.md b/packages/arista_ngfw/docs/README.md
index 032d22507..595d31f35 100755
--- a/packages/arista_ngfw/docs/README.md
+++ b/packages/arista_ngfw/docs/README.md
@@ -15,6 +15,7 @@ Arista NG Firewall supports several syslog output rules that may be configured o
 * Interface Stat Event
 * Intrusion Prevention Log Event
 * Session Event
+* Session Stats Event
 * System Stat Event
 * Web Filter Event

@@ -210,6 +211,7 @@ An example event for `log` looks as following:
 | event.dataset | Event dataset | constant_keyword |
 | event.duration | Duration of the event in nanoseconds. If event.start and event.end are known this value should be the difference between the end and start time. | long |
 | event.end | event.end contains the date when the event ended or when the activity was last observed. | date |
+| event.id | Unique ID to describe the event. | keyword |
 | event.ingested | Timestamp when an event arrived in the central data store. This is different from `@timestamp`, which is when the event originally occurred.  It's also different from `event.created`, which is meant to capture the first time an agent saw the event. In normal conditions, assuming no tampering, the timestamps should chronologically look like this: `@timestamp` \< `event.created` \< `event.ingested`. | date |
 | event.kind | This is one of four ECS Categorization Fields, and indicates the highest level in the ECS category hierarchy. `event.kind` gives high-level information about what type of information the event contains, without being specific to the contents of the event. For example, values of this field distinguish alert events from metric events. The value of this field can be used to inform how these kinds of events should be handled. They may warrant different retention, different access control, it may also help understand whether the data coming in at a regular interval or not. | keyword |
 | event.module | Event module | constant_keyword |
@@ -316,6 +318,7 @@ An example event for `log` looks as following:
 | source.geo.country_iso_code | Country ISO code. | keyword |
 | source.geo.country_name | Country name. | keyword |
 | source.geo.location | Longitude and latitude. | geo_point |
+| source.geo.postal_code | Postal code associated with the location. Values appropriate for this field may also be known as a postcode or ZIP code and will vary widely from country to country. | keyword |
 | source.geo.region_iso_code | Region ISO code. | keyword |
 | source.geo.region_name | Region name. | keyword |
 | source.ip | IP address of the source (IPv4 or IPv6). | ip |
```

File: packages/arista_ngfw/data_stream/log/fields/ecs.yml [hunk 1]
```diff
@@ -238,8 +240,12 @@
   name: source.geo.country_iso_code
 - external: ecs
   name: source.geo.country_name
+- external: ecs
+  name: source.geo.dma_code
```
Comment (efd6) [inline packages/arista_ngfw/data_stream/log/fields/ecs.yml:244]: This doesn't exist, so it can't be used.

Comment (MakoWish) [reply packages/arista_ngfw/data_stream/log/fields/ecs.yml:244]: Ugh... that is an artifact from some custom parsing with Logstash. The Logstash geoip filter plugin creates that field (or at least, it used to when I translated some "normalization" Logstash pipelines to Elasticsearch Ingest Pipelines). 

I'll get it removed.

File: packages/arista_ngfw/data_stream/log/fields/ecs.yml [hunk 2]
```diff
@@ -234,14 +234,14 @@
   name: source.domain
 - external: ecs
   name: source.geo.city_name
+- external: ecs
+  name: destination.geo.continent_code
```
Comment (efd6) [inline packages/arista_ngfw/data_stream/log/fields/ecs.yml:238]: This and the timezone addition should be removed.

Comment (MakoWish) [reply packages/arista_ngfw/data_stream/log/fields/ecs.yml:238]: Oh, I see. I had `destination.geo.continent_code` and `destination.geo.timezone` where they should have been `source.*`. I copy/pasted and forgot to rename to `source`. Those are valid ECS fields:

[geo.continent_code](https://www.elastic.co/guide/en/ecs/current/ecs-geo.html#field-geo-continent-code)
[geo.timezone](https://www.elastic.co/guide/en/ecs/current/ecs-geo.html#field-geo-timezone)

Comment (MakoWish) [reply packages/arista_ngfw/data_stream/log/fields/ecs.yml]: Updated.

**Review by efd6 — APPROVED** [review]

**Pull request merged**